### DISCUSSION TITLE
Some issues regarding a few of the new talents

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#13710] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The skill book Art of Submarine Warfare does not grant any weapons skill unless directly created by the War Stories talent. It's also unclear how much skill is gained per kill as characters with a low weapons skill seem to gain more skill points than characters with a higher weapons skill.

Machine Maniac and Quickfixer don't work on deconstructors, medical fabricators and duct blocks.
`tags="fabricator,door,engine,oxygengenerator,pump,turretammosource"`
Missing the `deconstructor`, `medicalfabricator` and `ductblock` tags.

Very minor discrepancy between Junction Junkie and Better Than New, Junction Junkie uses tags `tags="junctionbox,supercapacitor,battery,reactor,sonarmonitor,statusmonitor,navterminal"` while Better Than New uses `category="Electrical"` which doesn't include the reactor as it is not categorized as an electrical item.

Junction Junkie description states 500 XP while the actual amount given is 600.
```
  Description tag="talentdescription.junctionjunkie.30">
  <Replace tag="[amount]" value="10" color="gui.green"/>
  <Replace tag="[skill]" value="stattypenames.electricalskillbonus" color="gui.orange"/>
  <Replace tag="[xpamount]" value="500" color="gui.green"/>
</Description>
```
`<CharacterAbilityGiveExperience amount="600"/>`

These lines are somewhat vague as to whether or not there would be any benefit if those they were already at level 1
"-50: Upgrades the submarine's junction boxes to level 1."
"-50: Upgrades the submarine's pumps to level 1."
Currently if the sub is tier 2 and the junctions are at 1/2 upgrades then reaching the 50 repair threshold works and upgrades the junctions to level 2.

### Reproduction steps

1. Use Art of Submarine Warfare without the War Stories talent
2. Repair deconstructor, medical fabricator or duct block with the Machine Maniac talent

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Thanks for your report, the Machine Maniac and Quickfixer and minor discrepancy between Junction Junkie and Better Than New will all be addressed in the upcoming update. 

> The skill book Art of Submarine Warfare does not grant any weapons skill

This should also be addressed

> as characters with a low weapons skill seem to gain more skill points than characters with a higher weapons skill.

This is intended, most actions give less skill increase when players already have a high skill, but now this logic has also been applied to the War Stories talent item, however we'll work more on this to ensure it is more clear.

**Comment by JeroonBosch** [comment]
Closing the discussion, as we've created internal tickets to address the War Stories/Art of Submarine Warfare issue for a later update, and the other things should be addressed already.
